src/mesido/pycml/component_library/milp/electricity/electricity_base.py

```diff
@@ -13,8 +13,8 @@ def __init__(self, name, **modifiers):
         super().__init__(name, **modifiers)
 
         self.add_variable(Variable, "Power")
-        self.add_variable(Variable, "V", min=0.0)
         self.add_variable(Variable, "I")
+        self.add_variable(Variable, "V", min=0.0)
```

Comment:
add comment on the reason for this order of P, I and V

Reply:
actually the order doesn't matter any more in the current implementation